consensus/consensus_test.go

```diff
@@ -482,6 +482,11 @@ func TestUpdateConsensusOther(t *testing.T) {
 				"vote_unit_price":    "1",
 				"init_proposer": map[string]interface{}{
 					"1": []interface{}{"Y4TmpfV4pvhYT5W17J7TqHSLo6cqq23x3", "f3prTg9itaZY6m48wXXikXdcxiByW7zgk", "U9sKwFmgJVfzgWcfAG47dKn1kLQTqeZN3"},
+					"peerids": []interface{}{
```

Comment:
Should Peerids be included in round 1,
"1:", {addrs:{}, peerids:{}}

Reply:
good point.